helpers/openaiInstance.js

```diff
@@ -0,0 +1,29 @@
+const OpenAI = require('openai')
+const { observeOpenAI } = require('langfuse')
+const { 
+    openaiAPIKey, 
+    langfusePublicKey, 
+    langfuseSecretKey,
+    langfuseBaseURL
+} = require('../configs/app.config')
+
+const openaiClient = new OpenAI({
+    apiKey: openaiAPIKey,
+})
```

Comment:
Can we please instantiate this before the server starts? Please refer ASESSMENT-API to see how we have instantiated redis and node-cache